src/ResourceManager/RedisCache/Commands.RedisCache/Commands/SetAzureRedisCache.cs

```diff
@@ -62,6 +59,9 @@ public class SetAzureRedisCache : RedisCacheCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "The number of shards to create on a Premium Cluster Cache.")]
         public int? ShardCount { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "A hash table which represents tags.")]
```

Comment:
Can you set Zones? Just curious, if not this is fine.

Reply:
No, can't change zones